README.md

```diff
@@ -141,6 +145,10 @@ The following apps are supported by govuk-docker to some extent.
       * Rake task to [create a test taxon](https://github.com/alphagov/whitehall/blob/master/lib/tasks/taxonomy.rake#L11) for publishing is not idempotent
       * Placeholder images don't work as missing proxy for [/government/assets](https://github.com/alphagov/whitehall/blob/master/app/presenters/publishing_api/news_article_presenter.rb#L133)
 
+NB, there is an nginx-proxy service under `services/`, which is the only service
```

Comment:
I don't think we need this, as it's an implementation detail - we could also have other services for postgres, etc., if we needed to override config or something. Can we do without this line?

Reply:
I can remove this line 👍 better yet, would it make sense to move the `nginx-proxy` service to a separate directory to keep application config and the 'inner workings of govuk-docker' a little more separate from each other?

Reply:
Hmm...I guess there's a clash of concepts here. The 'services/' directory refers to docker-compose services (vs. a govuk-docker 'service' you can run/startup). I was going to say that's mostly an implementation detail, but then I see you can do this:

```
finder-frontend$guidance-links gdr --service nginx-proxy
Unknown stack: lite.

Available stacks:

app
```

While that's not great, in practice I doubt people will be browsing the services directory for things to run. I'd suggest waiting to see if others find this confusing before we introduce another place for config. How does that sound?

Reply:
Have removed 👍